README.md

```diff
@@ -1,8 +1,151 @@
-# sindri-scroll-sdk
+# Sindri Provers for Scroll SDK
 
-## Temporary Usage Instructions
 
-First, create your `config.json` file from a template
+<img src="./media/sindri-gradient-logo.webp" height="160" align="right"/>
+
+#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/) | [Getting Started](#getting-started) | [Development](#development)
+
+Sindri provides automated ZK proving infrastructure, empowering hundreds of teams — including leading Layer 2s and rollups — to launch in minutes instead of months.
+Through our API, developers can seamlessly integrate verifiable computation, reducing time to market, cutting costs, and scaling faster.
+Sindri makes zero-knowledge infrastructure simple and accessible, facilitating automation across every layer of the ZK app deployment stack.
+
+This repository hosts the images and Helm charts designed to perform proof generation for a Scroll SDK chain.
+The following diagram depicts how the Scroll SDK components work together when you use Sindri as a prover.
+```mermaid
+---
+title: Scroll SDK with Sindri Provers
+---
+flowchart LR
+    subgraph settle[Settlement Layer]
+        direction LR
+    end
+    subgraph sequence[Sequencing Layer]
+        direction LR
+    end
+    subgraph prove[Proving Layer]
+        direction LR
+        C["**Coordinator**"] -.->|Tasks| P@{ shape: processes, label: "**Provers**\nThese components outsource proof generation to Sindri's GPU workers" }
+        P -.->|Proofs| C
+    end
+    subgraph sindri["Scroll Circuits on Sindri"]
+        direction TB
+        Chunk["**Chunk**
+        Verified execution of contiguous blocks (i.e. a zkEVM proof)"]
+        Batch["**Batch**
+        Intermediate aggregation of multiple chunks"]
+        Bundle["**Bundle**
+        Final aggregate proof sent to the settlement layer"]
+    end
+    settle o--o sequence
+    sequence o--o prove
+    P --> Chunk
+    P --> Batch
+    P --> Bundle
+```
+
+For more information about the Sindri platform, please check out [sindri.app](https://sindri.app/).
+The next section will show you how to launch a local Scroll SDK devnet with provers running on Sindri.
+For production deployments, please consult the official [Scroll SDK documentation](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/production-deployment/).
+
+# Getting Started
+
+While this section primarily reiterates [Scroll's Devnet Guide](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/), it includes adjustments specifically for Ubuntu environments.
+This guide also makes revisions to launch the coordinator service which acts as a gateway between the sequencing layer and Sindri provers.
+Since the coordinator requires at least 20Gb of RAM, we recommend you use a machine with at least 32Gb available.
```

Comment:
Pretty sure convention is to separate the number from the units. Also, gigabyte is "GB".

```suggestion
Since the coordinator requires at least 20 GB of RAM, we recommend you use a machine with at least 32 GB available.
```